### DISCUSSION TITLE
Sorting Books via Chronological Order

### DISCUSSION THREAD

**Discussion by MakD** [Goodwy/PlayBooks#34] [Ideas]
If possible, can this be possible to sort books wrt it's chronological order. As an example, you can see in the attachment here from a different audiobook app![Screenshot_20230321-015016.png](https://user-images.githubusercontent.com/8217200/226456434-3d6deb0f-8998-42ac-a452-f3809c6bfee7.png)

Currently, the view available in playbooks is a bit randomised, i guess on the filename.
![Screenshot_20230321-015127.png](https://user-images.githubusercontent.com/8217200/226456718-adb0f94b-41bb-4b43-baa1-ee9f7294b867.png)

**Comment by MakD** [comment]
Also, somehow, playbook pulls a different name from what has been named in the folder structure, even after i tried to edit the tags. Can you help me sort that.

**Reply by MakD** [reply]
Or even having the option to arrange books by series would be a nice option. To have shelves with books in the same series arranged chronologically

**Reply by MakD** [reply]
@Goodwy any insights wrt to the book series implementation

**Reply by Goodwy** [reply]
The name of the book is taken not from the folder, but from the found files.
Sorting of books is in the plans.

**Reply by fbede** [reply]
@Goodwy please add grouping books into folders & collections

Edit: Thank you for your effort building this application
